File: Filler/Filler.API/Models/Receipt.cs
namespace Filler.API.Models
{
    public class Receipt
    {
        public  int ReceiptId { get; set; }

        public FuelType FuelType { get; set; } 

        public double Litres { get; set; }
        
        public DateTime FillTime { get; set; }

        public double Total { get; set; }

        public Pump UsedPump { get; set; }

        public string UserId { get; set; }

    }
}

File: Filler/ngFiller/src/app/models/receipt.ts
import { FuelType } from "./fuelType";
import { Pump } from "./pump";

export interface Receipt {
    id: number;
    fuelType: FuelType;
    litres: number;
    fillTime: string;
    total: number;
    usedPump: Pump;
    userId: string;
}
